modules/gitops-deploy-resources-different-namespaces.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deploy-resources-different-namespaces_{context}"]
+= Deploying resources to a different namespace
+
+To allow the Argo CD to manage resources in other namespaces apart from where it is installed, configure the target namespace with a `argocd.argoproj.io/managed-by` label.
+
+.Procedure
+* Configure the namespace:
```

Comment:
```suggestion
.Procedure

* Configure the namespace:
```

Hard to tell if this will cause any rendering issues without a preview, but I'd add a line space between headings and steps. Just a minor nit though and not a blocker if it doesn't cause rendering issues.